### DISCUSSION TITLE
An error occurred while rendering the visualization: TypeError: Cannot read properties of null (reading 'toString')

### DISCUSSION THREAD

**Discussion by bkowshik** [apache/superset#18737] [General]
## Reproduce bug

I am running a simple `GROUP BY` query like the one below and then do a Sunburst visualization.

```sql
SELECT
    DATE_PARSE(yyyymmdd, '%Y%m%d') yyyymmdd,
    city_name,
    COUNT(*) gross_orders
FROM orders
WHERE
    yyyymmdd = '20211012'
GROUP BY 1, 2
ORDER BY 3 DESC
```

## Expected results

I am expecting to see the visualization.

## Actual results

I am seeing the error below.

> An error occurred while rendering the visualization: TypeError: Cannot read properties of null (reading 'toString')

<img width="1680" alt="Screenshot 2022-02-15 at 6 47 28 AM" src="https://user-images.githubusercontent.com/2899501/153974945-b4bc4e17-d13f-4a93-8a95-adc0a0044039.png">

**Comment by zhaoyongjie** [comment]
Hi @bkowshik, It seems like you are using a very old version of Superset and lack of maintenance. I suggest that update to the current [stable version](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose).

**Comment by gokhangulumser** [comment]
i have the same problem. 
i use superset version : 1.4.0
what can i do?

![Ekran Alıntısı](https://user-images.githubusercontent.com/34714881/158132625-98369987-6119-4748-be68-6590622e8369.PNG)

**Reply by zhaoyongjie** [reply]
Hi there, Could you detail the issue, please?

**Reply by gokhangulumser** [reply]
hi, i added

**Reply by zhaoyongjie** [reply]
Which version do you use? How to reproduce this issue?

**Reply by gokhangulumser** [reply]
i use 1.4.0

sql lab -> sql editor -> write query and run -> explore -> save&explore.

![1](https://user-images.githubusercontent.com/34714881/158133834-533a86d4-4ac0-4e2f-93ec-521b23c45500.PNG)
![2](https://user-images.githubusercontent.com/34714881/158133839-1c9a1335-80a5-4cc1-974d-b4abecb3552c.PNG)
